code/modules/martial_arts/judo.dm

```diff
@@ -20,13 +20,30 @@
 	slot_flags = SLOT_FLAG_BELT
 	flags_2 = ALLOW_BELT_NO_JUMPSUIT_2
```

Comment:
This overrides the normal flag_2, which is `flags_2 = ALLOW_BELT_NO_JUMPSUIT_2 | BLOCKS_LIGHT_2`
Security belts don't override it, should this be removed?